meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-machine-id.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Machine-id Mount Point
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=systemd-machine-id-commit.service
+
+[Service]
```

Comment:
This was already addressed. The version is not pushed yet (lack of a laptop today)